### ISSUE TITLE
Add analytics use case

### ISSUE THREAD

**Issue by igrigorik** [w3c-webmob/netinfo-usecases#65]
From https://plus.google.com/+IlyaGrigorik/posts/QwTUPrcxyJp

![image](https://cloud.githubusercontent.com/assets/10652/3474050/06bc2294-02df-11e4-83c4-775bc6b4083e.png)

Exposing more detailed connection data (e.g. 2G, 3G, 4G), can help with analyzing performance data (RUM, etc), as well as business KPIs (slicing behavior, purchases, etc).

**Comment by marcoscaceres** [comment]
That's good feedback.  

As a kind of aside, it's worrying reading in the responses how many times media (images, audio) get a mention. It will be much more harmful to developers to delay images till DOMContentLoaded and rewriting all their image URLs to make use of net info (at least till we have SWs, but even then it will hurt on first load) ...  this could be a strong motivator to get a move on with client hints for those determined to serve content based on connection type. At least it could help with the URL rewriting.

**Comment by igrigorik** [comment]
@marcoscaceres right, a connection type CH hint would definitely help. I know of at least a few teams within Google that already do this sort of thing, albeit through much more painful means: matching IP blocks / looking at originating ASN, using cookies set on past navigations, etc. 

FWIW, Vivek did not mention it in his post, but they do make use of (and want) detailed cellular data.

**Comment by igrigorik** [comment]
Followup note... The analytics case should address 8.3: http://www.w3.org/TR/netinfo-usecases/#why-not-specifying-the-class-of-cellular-connection-2g-3g-etc.x
